### PR TITLE
Added sample of using custom giver

### PR DESCRIPTION
**Pull request by DezzK**

### CODE REVIEW

File: core-examples/node-js/custom-giver/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+* [Docker](https://docs.docker.com/desktop/#download-and-install) installed and running
+* [TONDEV CLI](https://docs.ton.dev/86757ecb2/p/179e51-tondev)
+
+
+## Preparation
+
+* [Run TON OS SE on your computer](https://docs.ton.dev/86757ecb2/p/19d886-ton-os-se) 
+
+```sh
+tondev se start
```
Comment (Artem-Zhdanov) [inline core-examples/node-js/custom-giver/README.md:17]: Now it is `everdev`

File: core-examples/node-js/custom-giver/index.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    });
+
+    // Wait until all messages have been produced:
+    const transactionTree = await client.net.query_transaction_tree({
+        in_msg: processingResult.transaction.in_msg,
+    });
+
+    // Make additional checks to ensure success:
+    if (transactionTree.transactions.length !== 2) {
+        throw `Something went wrong during requesting funds from the Giver: there must be 2 transactions, but actual count is ${transactionTree.transactions.length}`;
```
Comment (Artem-Zhdanov) [inline core-examples/node-js/custom-giver/index.js:85]: Line is too long, can it be made shorter ???

File: core-examples/node-js/custom-giver/index.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+                // Local node URL here
+                endpoints: ['http://localhost']
+            }
+        });
+        console.log("Custom Giver example");
+        await main(client);
+        process.exit(0);
+    } catch (error) {
+        if (error.code === 504) {
+            console.error(`Network is inaccessible. Check your connection. If you are using TON OS SE, make sure that you have started TON OS SE using \`tondev se start\`.\n If you run SE on another port or ip, replace http://localhost endpoint with http://localhost:port or http://ip:port in index.js file.`);
```
Comment (Artem-Zhdanov) [inline core-examples/node-js/custom-giver/index.js:163]: Line is too long

File: core-examples/node-js/custom-giver/index.js [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+    // See more info about ABI type here https://github.com/tonlabs/TON-SDK/blob/master/docs/mod_abi.md#abi
+    const abi = {
+        type: 'Contract',
+        value: HelloWallet.abi
+    }
+    // Generate an ed25519 key pair
+    const helloKeys = await client.crypto.generate_random_sign_keys();
+    
+    // Prepare parameters for deploy message encoding
+    // See more info about `encode_message` method parameters here https://github.com/tonlabs/TON-SDK/blob/master/docs/mod_abi.md#encode_message
```
Comment (Artem-Zhdanov) [inline core-examples/node-js/custom-giver/index.js:105]: Line is too long

File: core-examples/node-js/custom-giver/index.js [hunk 5]
```diff
@@ -0,0 +1,169 @@
+const { TonClient, abiContract, signerKeys, signerNone } = require("@tonclient/core");
```
Comment (Artem-Zhdanov) [inline core-examples/node-js/custom-giver/index.js:1]: signerNone unused var

**Review by Artem-Zhdanov — APPROVED** [review]

**Review by elasticLove1 — APPROVED** [review]

**Pull request merged**